app/assets/stylesheets/nfg_ui/network_for_good/core/nfg_theme/custom/_skeleton.scss

```diff
@@ -0,0 +1,50 @@
+// Skeleton styles for placheholder content
+.skeleton-avatar, .skeleton-heading, .skeleton-paragraph {
+  display: inline-block;
+  margin-bottom: ($spacer * .5);
+  background-color: $border-color;
+  overflow: hidden;
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: $body-bg;
+    animation: skeleton-sheen 1s linear infinite alternate;
+    content: '';
+
+    // any skeleton element inside a container with the the .loaded class removes the sheen animation.
+    .loaded & {
+      background-color: transparent;
+      animation: none;
+    }
+  }
+}
+.skeleton-avatar {
+  width: ($spacer * 1.5);
+  height: ($spacer * 1.5);
+  border-radius: 50%;
+}
+.skeleton-heading {
+  width: 70%;
+  height: $spacer;
+  border-radius: ($spacer * .5);
+  &:nth-child(even) { width: 45%; }
+}
+.skeleton-paragraph {
+  width: 100%;
+  height: ($spacer * .5);
+  border-radius: ($spacer * .25);
+  &:nth-child(2n+1) { width: 90%; }
+  &:nth-child(3n+1) { width: 75%; }
+  &:nth-child(4n+1) { width: 80%; }
+}
+
+// Sheen animation used on skeleton elements
+@keyframes skeleton-sheen {
+  0% { opacity: .45; }
+  to { opacity: .9; }
+}
```

Comment:
@kylejmorin if you want skeleton to be implemented as a pattern moving forward, we should probably document it and how to implement it on display-app

Reply:
Agreed.